docker/release/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM ubuntu:18.04
 
-RUN apt-get update
+RUN apt-get update; \
+    apt install -y software-properties-common; \
+    add-apt-repository ppa:ubuntu-toolchain-r/test
```

Comment:
can you try to add cleaning to this layer, eg ` && rm -rf /var/lib/apt/lists/*` ?